core/src/win32/filed/vss_generic.cc

```diff
@@ -534,6 +536,176 @@ bool VSSClientGeneric::Initialize(DWORD dwContext, bool bDuringRestore)
       errno = b_errno_win32;
       return false;
     }
+
+#  define VSS_CALL(Obj, Name, ...)                                 \
```

Comment:
Why is it now required to do an loop here?

Reply:
which loop ?  Do you mean the `do { ... } while (0)` construct ? Thats used to turn some code of block into a statement.  One should always do that for function like macros (if they arent statements by themselves already).

Otherwise the semicolon afterwards becomes optional.

Reply:
See for example here for reference: https://stackoverflow.com/questions/2687569/c-do-while0